sdks/python/apache_beam/options/pipeline_options_validator.py

```diff
@@ -231,6 +230,17 @@ def validate_cloud_options(self, view):
         errors.extend(self._validate_error(self.ERR_MISSING_OPTION, 'region'))
       else:
         view.region = default_region
+    dataflow_endpoint = view.dataflow_endpoint
+    if dataflow_endpoint is None:
+      errors.extend(
+          self._validate_error(self.ERR_MISSING_OPTION, dataflow_endpoint))
+    else:
+      valid_endpoint = validators.url(dataflow_endpoint)
+      print(dataflow_endpoint, validators.url(dataflow_endpoint))
```

Comment:
is this for debugging? if so, shall we remove it?

Reply:
Good catch, removed.